### ISSUE TITLE
Missing label for input select

### ISSUE THREAD

**Issue by te4-adrian-almetun-smeds** [adr29truck/APL-prep#2]
Looks like there's a label missing for this input. That makes it hard for people using screen readers or voice control to use the input.

_Originally posted by @accesslint[bot] in https://github.com/adr29truck/APL-prep/pull/1#discussion_r554476048_
